src/fides/api/util/tcf/tc_mobile_data.py

```diff
@@ -59,6 +65,9 @@ def _build_binary_string(name: str, num_bits: int) -> str:
         IABTCF_SpecialFeaturesOptIns=_build_binary_string(
             "special_feature_optins", SPECIAL_FEATURE_BITS
         ),
+        IABTCF_AddtlConsent=build_ac_string(tc_model.ac_vendor_consents)
+        if CONFIG.consent.ac_enabled
+        else None,
```

Comment:
This is where I exclude the AC strings from showing up in the mobile data if AC is not enabled